### ISSUE TITLE
For `list item` component, combine leading/trailing and text elements for Voice Over

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-ios#1715]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-ios/issues) for duplicate or closed feature requests

### Your test device

iPhone 17

### Device OS version

iOS 26

### Is Liquid Glass disabled

- [ ] Yes, Liquid Glass is disabled (or OS version is older than 26)
- [ ] No, Liquid Glass is enabled (or OS version is 26 or newer)

### OUDS iOS library version

3.0.0

### Describe the issue

Voice Over only read leading/trailing icon and image. I would suggest to read also flag, avatar and badge elements.
And to avoid multiple swipes, it would better to group leading/trailing element and text label.

Note : Slot element is read in two parts, the checkbox and the circle, combine both or set a custom accessibilityLabel on the whole.

### Expected Behavior

- All leading/trailing element should be vocalized
- Group leading/trailing elements and text label
- Keep reading order: leading-label-trailing

### Device settings

VO

**Comment by pya35** [comment]
same recommandations for `static` and `navigation` item
